CMakeLists.txt

```diff
@@ -166,9 +166,9 @@ else()
 endif()
 
 if(Boost_FIBER_LIBRARY_DEBUG)
-  get_filename_component(Boost_LIBRARY_DIR ${Boost_FIBER_LIBRARY_DEBUG} DIRECTORY)
+  get_filename_component(Boost_LIBRARY_DIR "${Boost_FIBER_LIBRARY_DEBUG}" DIRECTORY)
 elseif(Boost_FIBER_LIBRARY_RELEASE)
-  get_filename_component(Boost_LIBRARY_DIR ${Boost_FIBER_LIBRARY_RELEASE} DIRECTORY)
+  get_filename_component(Boost_LIBRARY_DIR "${Boost_FIBER_LIBRARY_RELEASE}" DIRECTORY)
```

Comment:
Sorry you are right the quotation marks are indeed not necessary here, and I agree that even if the code were accessible it wouldn't make a difference since we check in the if statement. I will remove these. Although on line 206-210 I think we need them, since `Boost_CONTEXT_LIBRARY_DEBUG` will only be set when cmake is found by the module mode

Reply:
> I will remove these. 

I'd suggest keeping them. I doubt it is common for `Boost_FIBER_LIBRARY_RELEASE` to have spaces or other nasty symbols in it, but I see no harm in handling it properly (at least here).